samples/e2eTestUtils/jest-puppeteer-utils/jestSetup.js

```diff
@@ -38,30 +38,18 @@ async function startServer(jestConfig) {
     }
 
     // Optional __TIMEOUT__ parameter or default to 30 seconds
-    const timeoutMs = jestConfig.globals.__TIMEOUT__ || 30000;
+    const timeoutMs = jestConfig.globals.__TIMEOUT__ || 60000;
```

Comment:
Does this help with the timeouts, or is this just a hope that double the time will make things more successful?  Would be good to understand what takes so long to complete here.

Reply:
Yes this should help with some of the timeouts we see. This is specifically for building and hosting the sample which can take a bit longer (for angular samples we even make this 90s - those take a long time to build)